pages/bare_metal_cloud/dedicated_servers/bring-your-own-linux/guide.en-gb.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Bring Your Own Linux (BYOLinux)
+excerpt: Find out how to easily deploy your own Linux images on dedicated servers
+updated: 2024-01-31
+---
+
+## Objective
+
+The Bring Your Own Linux feature (BYOLinux) enables you to deploy *cloudready* linux images directly on your dedicated server. You can therefore use the bare metal service as a resource for your deployments.
+
+**What does *cloudready* mean?**
+
+The *cloudready* standard generally means to be agnostic of the infrastructure on which the image is deployed.
+In addition to the prerequisites and limitations mentioned below, you must ensure that the image (downloaded or generated) answers correctly to the definition of technical expectations of a cloudready image.
+
+**This guide explains how to use BYOLinux on your OVHcloud dedicated server.**
+
+## Requirements
+
+- A [dedicated server](https://www.ovhcloud.com/en/bare-metal/) in your OVHcloud account
+- Access to the [OVHcloud Control Panel](https://ca.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/world/&ovhSubsidiary=we) (for the method "[Deployment via Control Panel](#viacontrolpanel)")
+- Access to the [OVHcloud API](/pages/manage_and_operate/api/first-steps) (for the section "[Deployment via API](#viaapi)" of this guide)
+- Your image must be smaller than the Server RAM minus 3GiB
+
+> [!warning]
+>
+> A new installation with BYOLinux will erase all the data on the server.
+>
+
+## Instructions
+
+**Technical limitations**
+
+There are some technical limitations linked to the use of physical products such as dedicated servers. Here is a non-exhaustive list, to keep in mind during your deployment preparation:
+
+- Boot type: **uefi** or **legacy**
+- Partition type: **MBR** or **GPT**
+- Image format: **qcow2**
+
+**Deployment methods**
+
+- [Deployment via the Control Panel](#viacontrolpanel): allows you to simply deploy your image using the OVHcloud Control Panel.
+- [Deployment via API](#viaapi): you can use the OVHcloud API to integrate images into your own scripts to automate deployments.
+
+### Deploy your image via the Control Panel <a name="viapanel"></a>
+
+***TBD***
```

Comment:
![image](https://github.com/ovh/docs/assets/17429937/2cfa8207-5d03-43b0-999c-0f19662ca246)
![image](https://github.com/ovh/docs/assets/17429937/9e89cf79-1539-478e-a7cd-8aba4c11b30c)

Reply:
You can use and adapt this text: https://github.com/ovh/docs/blob/fix/PUBM-35205-byoi/pages/bare_metal_cloud/dedicated_servers/bring-your-own-image/guide.en-gb.md?plain=1#L57-L75

Reply:
![2024-01-25_16-50](https://github.com/ovh/docs/assets/17429937/431c24ba-8fb2-4abb-bf87-7609afa16650)
![2024-01-25_16-53](https://github.com/ovh/docs/assets/17429937/45b40823-65fc-4017-b3da-f05c240c4aba)